charts/model-engine/values_sample.yaml

```diff
@@ -139,6 +142,8 @@ serviceTemplate:
 config:
   values:
     infra:
+      # cloud_provider [required]; either "aws" or "azure"
+      cloud_provider: aws
       # k8s_cluster_name [required] is the name of the k8s cluster
```

Comment:
some of the config values seem to be only used with aws, like `default_region`, `ml_account_id`, `s3_bucket`, sqs stuff. not need to be done in this PR, but we should move them into aws config section